src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceFacadeImpl.java

```diff
@@ -224,21 +225,22 @@ private String constructDetailedMessage(int totalRecords, int suspendedRecords,
         }
         if (uploadedRecords > 0 && suspendedRecords > 0) {
             detailedMessage = format(RECORDS_FAILED, failedRecords.size()) + DELIMITER_COMMA + SPACE
-                    + format(RECORDS_UPLOADED, uploadedRecords) + DELIMITER_COMMA + SPACE + AND + SPACE
-                    + format(RECORDS_SUSPENDED, suspendedRecords);
+                + format(RECORDS_UPLOADED, uploadedRecords) + DELIMITER_COMMA + SPACE + AND + SPACE
```

Comment:
if possible we can use concat method , its good to have.

Reply:
My bad. I should have done this when I pushed these changes. @abhijitdiwan feel free to address this comment or else I shall do as part of my ongoing ticket :)